temporalcli/commandsmd/commands.md

```diff
@@ -560,533 +1300,836 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
-* `--task-queue`, `-t` (string) - Name of the Task Queue. Required.
+* `--build-id` (string) -
+  Build Id within the promoted set.
+  Required
+* `--task-queue`, `-t` (string) -
+  Task Queue name.
+  Required.
 
+### temporal workflow: Start, list, and operate on Workflows
 
-### temporal workflow: Start, list, and operate on Workflows.
+[Workflow](/workflows) commands perform operations on [Workflow Executions](/workflows#workflow-execution):
 
-[Workflow](/concepts/what-is-a-workflow) commands perform operations on [Workflow Executions](/concepts/what-is-a-workflow-execution).
+```
+temporal workflow [command] [options]
+```
 
-Workflow commands use this syntax: `temporal workflow COMMAND [ARGS]`.
+For example:
+
+```
+temporal workflow list
+```
 
 #### Options set for client:
 
-* `--address` (string) - Temporal server address. Default: 127.0.0.1:7233. Env: TEMPORAL_ADDRESS.
-* `--namespace`, `-n` (string) - Temporal server namespace. Default: default. Env: TEMPORAL_NAMESPACE.
-* `--api-key` (string) - Sets the API key on requests. Env: TEMPORAL_API_KEY.
-* `--grpc-meta` (string[]) - HTTP headers to send with requests (formatted as key=value).
-* `--tls` (bool) - Enable TLS encryption without additional options such as mTLS or client certificates. Env:
-  TEMPORAL_TLS.
-* `--tls-cert-path` (string) - Path to x509 certificate. Env: TEMPORAL_TLS_CERT.
-* `--tls-key-path` (string) - Path to private certificate key. Env: TEMPORAL_TLS_KEY.
-* `--tls-ca-path` (string) - Path to server CA certificate. Env: TEMPORAL_TLS_CA.
-* `--tls-cert-data` (string) - Data for x509 certificate. Exclusive with -path variant. Env: TEMPORAL_TLS_CERT_DATA.
-* `--tls-key-data` (string) - Data for private certificate key. Exclusive with -path variant. Env: TEMPORAL_TLS_KEY_DATA.
-* `--tls-ca-data` (string) - Data for server CA certificate. Exclusive with -path variant. Env: TEMPORAL_TLS_CA_DATA.
-* `--tls-disable-host-verification` (bool) - Disables TLS host-name verification. Env:
-  TEMPORAL_TLS_DISABLE_HOST_VERIFICATION.
-* `--tls-server-name` (string) - Overrides target TLS server name. Env: TEMPORAL_TLS_SERVER_NAME.
-* `--codec-endpoint` (string) - Endpoint for a remote Codec Server. Env: TEMPORAL_CODEC_ENDPOINT.
-* `--codec-auth` (string) - Sets the authorization header on requests to the Codec Server. Env: TEMPORAL_CODEC_AUTH.
-
-### temporal workflow cancel: Cancel a Workflow Execution.
-
-The `temporal workflow cancel` command is used to cancel a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+* `--address` (string) -
+  Temporal Service gRPC endpoint.
+  Default: 127.0.0.1:7233.
+  Env: TEMPORAL_ADDRESS.
+* `--namespace`, `-n` (string) -
+  Temporal Service Namespace.
+  Default: default.
+  Env: TEMPORAL_NAMESPACE.
+* `--api-key` (string) -
+  API key for request.
+  Env: TEMPORAL_API_KEY
+* `--grpc-meta` (string[]) -
+  HTTP headers for requests.
+  Format as "KEY=VALUE" pairs.
+* `--tls` (bool) -
+  STOPPED HERE WEFWEF
```

Comment:
ACK; I've made it this far in my review comments. Stopping here as well.

Reply:
Woo! Making more changes for you!